src/utils/time.ts

```diff
@@ -182,10 +213,37 @@ export const roundToNearestHour = (seconds: number) => {
   return Math.floor(seconds / oneHour) * oneHour;
 };
 
+/**
+ * Format UTC seconds as a string representing weekday abbreviation
+ *
+ * @example
+ *   formatDateToWeekday(1618509600)
+ *   // 'Thu'
+ *
+ * @param utcSeconds
+ *
+ * @see https://date-fns.org/docs/format
+ */
 export function formatDateToWeekday(utcSeconds: number) {
   return format(new Date(utcSeconds * ONE_SECOND_IN_MILLISECONDS), "E");
 }
 
+/**
+ * Format UTC seconds as a string representing relative date
+ *
+ * @example
+ *   formatUtcSecondsRelativeToToday(1618509600)
+ *   // 'today at 9:00 PM'
+ *
+ * @param utcSeconds
+ *
+ * @see https://date-fns.org/v2.21.1/docs/formatRelative
+ */
+export const formatUtcSecondsRelativeToToday = (utcSeconds: number) => {
+  const date = new Date(utcSeconds * ONE_SECOND_IN_MILLISECONDS);
+  return formatRelative(date, new Date());
+};
```

Comment:
It looks like this function accepts a `Date` or a `Number`:

- https://date-fns.org/v2.21.1/docs/formatRelative#arguments

Is that `Number` in seconds or milliseconds? We could probably avoid 2 unnecessary object allocations here. Something like:

```suggestion
export const formatUtcSecondsRelativeToToday = (utcSeconds: number) => {
  const utcMilliseconds = utcSeconds * ONE_SECOND_IN_MILLISECONDS;
  return formatRelative(utcMilliseconds, Date.now());
};
```

Reply:
And if that works, can we also do similar with `format` in the other functions in this file? It seems to accept `Date` / `Number` as well:

- https://date-fns.org/v2.21.1/docs/format#arguments

Reply:
refactored some functions here: https://github.com/sparkletown/sparkle/pull/1328/commits/bec9ea233a7130a836079592bff2df7aae3a3734